src/pools/D3MAaveDaiPool.sol

```diff
@@ -0,0 +1,212 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Copyright (C) 2021-2022 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity >=0.6.12;
+
+import "./ID3MPool.sol";
+
+interface TokenLike {
+    function approve(address, uint256) external returns (bool);
+    function transfer(address, uint256) external returns (bool);
+    function transferFrom(address, address, uint256) external returns (bool);
+    function balanceOf(address) external view returns (uint256);
+}
+
+interface CanLike {
+    function hope(address) external;
+    function nope(address) external;
+}
+
+interface D3mHubLike {
+    function vat() external view returns (address);
+}
+
+interface ATokenLike is TokenLike {
```

Comment:
this is what we wanted, correct? e71b6c9

Reply:
usually it's a link to github source code as it's also done in the compound branch.

Reply:
I would prefer to either link to a github source code at a specific commit OR etherscan, I wouldn't want to link to the main branch code.

In this case I think linking to etherscan is a little better since we can link to the address we are using (a Proxy) and see both the proxy and the implementation.  But that is just the reason I went this way, let me know if you feel strongly

Reply:
yeah, sounds good.